backend/controllers/social.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+import socialModel, { ISocial } from '../models/social.model';
```

Comment:
Could we use absolute path?

Reply:
I think it requires some additional package when Weusing ts.

Reply:
How is this working in auth repository?

Reply:
@hendrysiak